src/app/modules/time-clock/components/entry-fields/entry-fields.component.html

```diff
@@ -1,4 +1,4 @@
-<form [formGroup]="entryForm" (ngSubmit)="onSubmit()">
+<form [formGroup]="entryForm">
```

Comment:
I am concerned about this change. As I remember, the line you are removing, is in charge to update automatically the time entry, each time the user updates any other field, different from time in. If you remove this line, you remove that behavior, and user will have to explicitly clock out to save data. Make sure, that such behavior remains or is handled in another way.

Reply:
I also thought the same when I removed that line, but then I realized that each field has (blur) that calls the onSubmit method. It is for this reason that in my previous ticket, it did not follow the correct order, because it ran onSubmit twice.

![Captura de pantalla de 2020-12-10 15-58-02](https://user-images.githubusercontent.com/73141380/101829119-8574df80-3b00-11eb-805e-5be02dae35ec.png)

I already tried it updating all the fields and these are perfectly updated because when clicking it runs the onSubmit method

Reply:
If we are sure that what @Angeluz-07 is not happening, go ahead. I insist that putting FF (Feature Toggle) would help to solve the problem.

Reply:
@PaulRC-ioet  it's ok buddy if you checked it. Just wanted to highlight that.

Reply:
Sure, I tried it again and it didn't give me an error @josepato87 @Angeluz-07